### DISCUSSION TITLE
[v4] How can i center a container by default

### DISCUSSION THREAD

**Discussion by sharathdaniel** [tailwindlabs/tailwindcss#15429] [Help] [answered]
In v3, i can set the option in config like this: 
` theme: {
    container: {
      center: true,
      padding: '0.9375rem' /* 15px */
    }
    }`

Do i need to use `mx-auto` class now?

**Comment by sharathdaniel** [comment, marked as answer]
Got the answer from [here](https://tailwindcss.com/docs/v4-beta#configuring-the-container-utility)

**Reply by 1w3j** [reply]
it actually is explained and updated in [here now](https://tailwindcss.com/docs/upgrade-guide#container-configuration)
